docs/schemas.md

```diff
@@ -0,0 +1,17 @@
+<style>
+div.doc-function {
+    display: none;
+}
+</style>
+
+# Schemas
+
+::: helm.benchmark.scenarios.scenario.Scenario
+::: helm.benchmark.adapter.RequestState
+::: helm.benchmark.scenarios.scenario.Instance
+::: helm.benchmark.scenarios.scenario.Reference
+::: helm.benchmark.augmentations.perturbation_description.PerturbationDescription
+::: helm.common.request.Request
+::: helm.common.request.RequestResult
+::: helm.benchmark.metrics.metric.PerInstanceStats
+::: helm.benchmark.metrics.statistic.Stat
```

Comment:
What about things like `MetricName` and `Run`, and `AdapterSpec`?

Reply:
Looks like we need to add a lot more stuff like `ScenarioState`... I'll merge this and add a todo in #1159

Reply:
Could we automatically grab any class that has a `@dataclass` annotation?